### DISCUSSION TITLE
PR review requests

### DISCUSSION THREAD

**Discussion by berchn** [tree-sitter/tree-sitter#2256] [Q&A]
Is there a process for requesting reviews of PRs? In particular, I'm interested in the feature set in https://github.com/tree-sitter/tree-sitter-cpp/pull/173, but it's been untouched for nearly a year by any of the maintainers of https://github.com/tree-sitter/tree-sitter-cpp.

**Comment by ahlinc** [comment]
I'd like to implement such flow in some future with a check list of PR requirements that would help for maintainers to review and merge commits for languages where maintainers aren't excellent experts. But contributors would require to provide more supportive info about changes. That's just my view of the path to resolve the problem.

For now I'd suggest to make a fork and merge there what you want.
